extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_links.php

```diff
@@ -18,7 +18,21 @@
 		<div class="wds-dropdown__content wds-global-navigation__dropdown-content <?= $dropdownRightAligned ? 'wds-is-right-aligned' : ''; ?>">
 			<ul class="wds-list">
 				<?php foreach ( $model['links'] as $link ): ?>
-					<li><?= $app->renderView( 'DesignSystemGlobalNavigationService', 'linkDropdown', [ 'model' => $link ] ); ?></li>
+					<li>
+						<?php if ( $link['type'] === 'link-authentication' ) {
+							echo $app->renderView(
+								'DesignSystemGlobalNavigationService',
+								'linkAuthentication',
+								[ 'model' => $link ]
+							);
+						} else {
+							echo $app->renderView(
```

Comment:
in if..else you have nearly the same code. I propose to have variable for `linkDropdown` or `linkAuthentication` and then after if call render